### PR TITLE
Create & edit events

### PR DESCRIPTION
**Pull request by munteanumarco**
User can now create and edit events.
Integrated with Google Maps API for the location management.

### CODE REVIEW

File: event-planner-frontend/src/features/event/components/fields/SkillLevelField.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { FormControl, FormLabel, Select } from '@chakra-ui/react';
+import { SkillLevel } from 'features/event/api/models';
+
+interface SkillLevelFieldProps {
+  skillLevel: number;
+  onChange: (e: any) => void;
```
Comment (Filip-B-32) [inline event-planner-frontend/src/features/event/components/fields/SkillLevelField.tsx:7]: is good practice for an event to have the form event type, not any

**Review by PhoenixItachi — APPROVED** [review]

File: event-planner-frontend/src/features/event/components/fields/PositionSelection.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Box, FormLabel, Select } from '@chakra-ui/react';
+import { GetPositionForSportTypeDto } from 'features/event/api/dtos';
+import { EventPosition } from 'features/event/api/models';
+import IndividualAvailablePositions from './IndividualAvailablePositions';
+
+interface PositionSelectionProps {
+  positionsForSportType: GetPositionForSportTypeDto[];
+  handlePositionChange: (e: any) => void;
```
Comment (Filip-B-32) [inline event-planner-frontend/src/features/event/components/fields/PositionSelection.tsx]: it's good practice to put form event type, not any

File: event-planner-frontend/src/features/event/components/fields/MaxParticipantsField.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { FormControl, FormLabel, Input } from '@chakra-ui/react';
+
+interface MaxParticipantsFieldProps {
+  maximumParticipants: number;
+  onChange: (e: any) => void;
```
Comment (Filip-B-32) [inline event-planner-frontend/src/features/event/components/fields/MaxParticipantsField.tsx:6]: same here

File: event-planner-frontend/src/features/event/components/fields/EventNameField.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { FormControl, FormLabel, Input } from '@chakra-ui/react';
+
+interface EventNameFieldProps {
+  eventName: string;
+  onChange: (e: any) => void;
```
Comment (Filip-B-32) [inline event-planner-frontend/src/features/event/components/fields/EventNameField.tsx]: same here

**Review by HellHound97 — APPROVED** [review]

**Review by HellHound97 — APPROVED** [review]

**Review by alexandratodea — APPROVED** [review]

**Pull request merged**